.circleci/config.yml

```
@@ -42,16 +50,32 @@ jobs:
       - run:
           name: build book
           command: just build
+
   lint-links:
-    machine:
-      image: <<pipeline.parameters.base_image>>
+    executor: base
     steps:
       - checkout
       - run:
           name: Lint check
           command: just lint-links
       - notify-failures-on-develop:
           channel: C055R639XT9 #notify-link-check
+
+  publish-pages:
+    executor: builder
```

```suggestion
    executor: base
```
ethereum-optimism/circleci-utils works best with ubuntu machine.

---

Oh no this one needs to be the `builder` since it needs `cargo`